catroid/src/main/java/org/catrobat/catroid/ui/MyProjectsActivity.java

```diff
@@ -153,4 +153,13 @@ private void handleShowDetails(boolean showDetails, MenuItem item) {
 
 		item.setTitle(showDetails ? R.string.hide_details : R.string.show_details);
 	}
+
+	@Override
+	public void onBackPressed() {
+		if (projectListFragment != null) {
+			projectListFragment.cancelLoadProjectTask();
+			Log.d(TAG, "Back press detected!");
+		}
```

Comment:
projectListFragment is always set, no null check needed
Please remove log output